mmpose/datasets/datasets/mesh/mesh_mix_dataset.py

```diff
@@ -0,0 +1,40 @@
+from abc import ABCMeta
+
+import numpy as np
+from torch.utils.data import Dataset
+
+from mmpose.datasets.builder import DATASETS
+from .mesh_base_dataset import MeshBaseDataset
+
+
+@DATASETS.register_module()
+class MeshMixDataset(Dataset, metaclass=ABCMeta):
+    """Mix Dataset for 3D human mesh estimation.
+
+    The dataset combines data from multiple datasets (MeshBaseDataset) and
+    sample the data from different datasets with the provided proportions.
+    The dataset loads raw features and apply specified transforms
+    to return a dict containing the image tensors and other information.
+
+    Args:
+        configs (list): List of configs for multiple datasets.
+        partition (list): Sample proportion of multiple datasets.
```

Comment:
give an example of the partition. if they are required to be sum to 1, then make this requirement explicit.

Reply:
and also check it by assert

Reply:
OK, I get  it.